cli.py

```diff
@@ -56,6 +59,16 @@ def startserver(adduser: Tuple, api: str, auth: bool, dburl: str,
     # DB_URL = 'sqlite:///database.db'
     DB_URL = dburl
 
+    if openapi.name != "doc.jsonld":
+        with open(openapi.name, 'r') as stream:
+            try:
+                openapi_doc = yaml.load(stream)
+            except yaml.YAMLError as exc:
+                print(exc)
+        api_doc = parse(openapi_doc)
+        f = open("./hydrus/samples/hydra_doc_sample.py", "w")
```

Comment:
Why do you need to save api_doc? Just use this parsed doc to start the server.
Ideally, there should be two commands one to convert and save openapi doc to hydradoc and other to directly serve using openapi doc.

Reply:
the api doc is string , like [this](https://github.com/HTTP-APIs/hydrus/blob/develop/hydrus/samples/hydra_doc_sample.py) , i need the doc object from the api doc thats why i saved it and later used the doc imported .

Reply:
Use the parse function from the openapi_parser then

Reply:
thats what im calling in line 68 right ?

Reply:
Okay, it's fine then but I you get some time there should be a function to return the doc in proper format not the string then this function can later be used to create JSON dump and string format to store the doc in a file.

Reply:
yes makes sense ill do that for sure